### PR TITLE
Update package manager installs to Python 3.

### LINKED ISSUE #14656: Need to update Python interpreter for Docker CI/CD.
## Bug Description

Since the framework is now purely Python 3, I need to do some housekeeping, because both the Ubuntu and CentOS variants of the MOOSE Docker image have Python 2 as the default interpreter.

## Steps to Reproduce

Running `./run_tests` when the default interpreter _hasn't_ been updated to Python 3 results in a stack trace, since the shebang is
```
#!/usr/bin/env python
```

## Impact

Marginal.

### PR DESCRIPTION
**Pull request by herter4171**
* Changed package installs to Python 3 for `apt` and `yum`
* Set default Python interpreter to 3 via
  * `update-alternatives` for `apt`
  * Symbolic link for `yum`
* Tests run as they should for both distros, so closes #14656

### CODE REVIEW

**Comment by herter4171** [comment]
Hi @permcody, I hate to be the squeaky wheel, but could you or someone else on the team take a look at this real quick?  Assigning a reviewer seems bossy, but maybe that's just me?  Thanks.

**Comment by permcody** [comment]
@aeslaughter or @milljm - Can one of you look at this and see if this is the right fix for Justin's issue?

File: docker_ci/apt_installs.sh [hunk 1]
```diff
@@ -32,3 +31,6 @@ apt-get install -y \
 
 # Clear cache
 rm -rf /var/lib/apt/lists/*
+
+# Set Python 3 as the default interpreter
+update-alternatives --install /usr/bin/python python /usr/bin/python3 10
```
Comment (dschwen) [inline docker_ci/apt_installs.sh]: This is what I had to issue manually on my apt based installs. Looks good to me.

**Review by milljm — APPROVED** [review]
Looks good to me as well.

**Pull request merged**